_posts/zh_CN/releases/2016-10-27-release-0.13.1.md

```diff
@@ -99,7 +99,7 @@ Segwit 软分叉的激活是用 BIP9 的 versionbits 来实施的。Segwit 的 v
 日启动之后，节点们就会在 2016 个区块算力重新调整期的开始的时候追踪区块支持 Segwit 的信号。当这两个礼拜的重新调整期内，有
 95%的区块发出信号支持 Segwit，这个软分叉就会被锁定。然后过了另外 2016 个区块后，Segwit 就会被激活。 
 
-如果你想获得关于 Segwit 的更多信息，请参阅 [Segwit FAQ][]，和 Segwit [钱包开发者指导手 册][]  或是 [141][BIP141], [143][BIP143],
+如果你想获得关于 Segwit 的更多信息，请参阅 [Segwit FAQ][]，和 Segwit [钱包开发者指导手 册][segwit wallet developers guide]  或是 [141][BIP141], [143][BIP143],
```

Comment:
I believe the text for the first link here should be 隔离见证的好处 rather than 'Segwit FAQ' and it should link to https://bitcoincore.org/zh_CN/2016/01/26/segwit-benefits/ rather than the English page.

Reply:
That wasn't a link changed in this PR, but I agree it would be an improvement.  Commit added to PR, thanks!